src/types/Base.js

```diff
@@ -25,9 +25,10 @@ export default class Base extends EventEmitter {
    *				                 gets the Update object containing message
    *
    * @param  {object} bot
+   * @param {boolean} expectAnswer whether a sent message expects an answer from a contact(s)
    * @return {Promise} returns a promise, resolved with message:answer
    */
-  send(bot) {
+  send(bot, expectAnswer = false) {
```

Comment:
This is a breaking change, I would default to `true`, and in your use case, you could avoid unnecessary listeners by setting the value to false explicitly.

It might actually be the right thing to do to only listen when explicitly asked, but since this is a breaking change, I'd say we choose the safe path for now. If necessary, a major release can break this.